content/zh/docs/faas/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: 功能即服务（Faas）
+linkTitle: FaaS
+description: >-
+  OpenTelemetry 支持不同云供应商提供的各种功能即服务（Faas）的监控方法
```

Comment:
```suggestion
  OpenTelemetry 支持使用各种方法来监控不同云供应商提供的功能即服务（Faas）
```

Reply:
I feel that this suggestion deviates from the original intention.

Reply:
@reyang WDYT?

Reply:
> I feel that this suggestion deviates from the original intention.

+1

Reply:
OK, let's keep original and mark it resolved